### ISSUE TITLE
TableView improvements 

### ISSUE THREAD

**Issue by tznind** [gui-cs/Terminal.Gui#2543]
I've split this ticket into 3:

- [x] #2551
- [x]  #2553
- [ ] Option to draw lines to a `LineCanvas`

-------------------

* Not a fan of the selection extending to the border.
* Why is no bottom border drawn?

If there's no point to having the cursor flashing, I say turn it off.

Now that the new `Border` is in place (and `LineCanvas` kicks even more ass than before), it'd be great of all these lines connected with minimal screen real estate being used.

<img width="477" alt="image" src="https://user-images.githubusercontent.com/585482/232048379-01a5a243-affc-439c-8fbd-44457692bce1.png">

You may want to wait on that until `TileView` is REALLY not needed anymore, which is dependent on #2483, #2537,

_Originally posted by @tig in https://github.com/gui-cs/Terminal.Gui/issues/2522#issuecomment-1508460746_

**Comment by tznind** [comment]
> If there's no point to having the cursor flashing, I say turn it off.

Have moved this to a new issue since its largely to do with `TableView`.

**Comment by tznind** [comment]
Flashing cursor in a specific column is important for horizontal navigation (e.g. when user cannot see all columns at once).  TableView has a single selected cell which drives navigation.  If all the columns are visible and there is full row selection enabled this can feel pointless but in the general case its usually important.
